pkg/test/env.go

```diff
@@ -42,8 +44,9 @@ type (
 )
 
 var (
-	allRegisterControllerFuncs = make(map[string]registerControllerFunc, 0)
-	allRegisterWebhookFuncs    = make(map[string]registerWebhookFunc, 0)
+	allRegisterControllerFuncs   = make(map[string]registerControllerFunc, 0)
+	allRegisterWebhookFuncs      = make(map[string]registerWebhookFunc, 0)
+	useExistingGreenhouseCluster = clientutil.GetEnvOrDefault("TEST_USE_EXISTING_GREENHOUSE_CLUSTER", "false") == "true"
```

Comment:
Can we use `USE_EXISTING_CLUSTER` directly? AFAICS, we are using this env var to set the value for the other env var.

Reply:
:100: